### PR TITLE
Add rust polars version info

### PR DESCRIPTION
**Pull request by sorhawell**
this is an R side implementation. A rust side implementation will likely be available from rust-polars 0.31.0
https://github.com/pola-rs/polars/pull/9660

This partially resolves: https://github.com/pola-rs/r-polars/issues/288

### CODE REVIEW

File: R/info.R [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# get version of rust-polars version from Cargo.lock at package build time
+# from polars 0.31.1 this can be migrated to rust side see
+# https://github.com/pola-rs/polars/pull/9660
+RUST_POLARS_VERSION = (\() {
+  Cargo.lock = readLines("./src/rust/Cargo.lock")
```
Comment (eitsupi) [inline R/info.R:5]: I don't think this will be working user side.

**Review by eitsupi — CHANGES_REQUESTED** [review]
Thanks, but this is needed and will be working?

File: R/info.R (hunk 1 above)
Comment (sorhawell) [reply R/info.R:5]: I think it would work. In what scenario? If all files are deleted after compilation?

Comment (sorhawell) [reply R/info.R:5]: It happens at package build time.

Comment (etiennebacher) [reply R/info.R:5]: just tried and I can confirm it works:

``` r
library(polars)
list.files(".")
#> [1] "ok-coqui_reprex.R"        "ok-coqui_reprex.spin.R"  
#> [3] "ok-coqui_reprex.spin.Rmd"
pl$polars_info()
#> r-polars package version : 0.7.0.9000
#> rust-polars crate version: 0.30.0
#> 
#> Features:          
#> simd FALSE
```
(also works in another project directory)

Comment (eitsupi) [reply R/info.R:5]: Ok, thanks.

**Review by eitsupi — APPROVED** [review]

**Pull request merged**